src/test/java/amazonTestSuite/amazonFeaturesTest.java

```diff
@@ -44,6 +44,7 @@ public void authorsNameTest(){
         homePage.searchBySearchingBarByText(testName);
 
         CategoryByBooksPage categoryByBooksPage = new CategoryByBooksPage(driver);
+        categoryByBooksPage.getPage("s?k=Tsugumi+Ohba&i=stripbooks-intl-ship&ref=nb_sb_noss");
```

Comment:
Remind me talk about this later. This shouldn't be so, but don't fix this for now.

Reply:
Ok